cob_bringup/robots/cob4-1.launch

```diff
@@ -5,17 +5,17 @@
 	<arg name="pkg_env_config" default="$(find cob_default_env_config)"/>
 
 	<include file="$(find cob_bringup)/robots/cob4-1.xml" >
-		<arg name="cob4-1-b1" value="cob4-1-b1"/>
-		<arg name="cob4-1-t1" value="cob4-1-t1"/>
-		<arg name="cob4-1-t2" value="cob4-1-t2"/>
-		<arg name="cob4-1-t3" value="cob4-1-t3"/>
-		<arg name="cob4-1-s1" value="cob4-1-s1"/>
-		<arg name="cob4-1-h1" value="cob4-1-h1"/>
+		<arg name="cob4-1-b1" value="cob4-2-b1"/>
+		<arg name="cob4-1-t1" value="cob4-2-t1"/>
+		<arg name="cob4-1-t2" value="cob4-2-t2"/>
+		<arg name="cob4-1-t3" value="cob4-2-t3"/>
+		<arg name="cob4-1-s1" value="cob4-2-s1"/>
+		<arg name="cob4-1-h1" value="cob4-2-h1"/>
```

Comment:
this is required as we run the cob4-1 setup on the cob4-2 hardware and the PCs are named like that